_posts/2022-05-17-release-1.0.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: Launching FocusCuts
+date: 2022-05-17 17:00
+author: Rosemary Orchard
```

Comment:
The only reason why I added my name is because I had something about podcasting with David Sparks and contextual computing in there, we can ditch the author!

Reply:
I don't see the bit about podcasting (but that could be because GitHub on iOS is not the most easy app to use) but I don't think the contextual clause changes the authorship that much?  It's still pretty easy / obvious that you are a co-founder and you'll be making the Discord posts, talking about it on podcasts etc. that then help people know the connection?